py3status/modules/volume_status.py

```diff
@@ -149,6 +143,97 @@ def current_volume(self):
         # get info about channel mute status
         muted = self._get_muted(output)
 
+        return perc, muted
+
+    def volume_up(self, delta):
+        # volume up
+        cmd = 'amixer -q -D {} sset {} '.format(self.device, self.channel)
```

Comment:
That was the original code from volume_status so I didn't change it. Will do.

@tobes: I don't think they should return command because hypothetically, a new backend could not have a `toggle` command and would need to execute two commands, one to get the status and one to set the new status. This is just an example, but this is why I think the backend classes should do the interaction.